routes/orderProducts.js

```diff
@@ -0,0 +1,48 @@
+const orderProductsRouter = require("express").Router();
+const { getOrderProductsById, addProductToOrder, updateOrderProduct, destroyOrderProduct} = require("../db")
+// const { isLoggedIn, isAdmin } = require("./util") need?
+
+
+orderProductsRouter.post("/orders/:orderId/products",  async (req, res, next) => {
+    const { orderId } = req.params;
+    const { productId, price, quantity } = req.body;
+    try {
+        const order_product = await addProductToOrder(orderId, productId, price, quantity)
```

Comment:
`addProductToOrder` should take one parameter that is an object: `addProductToOrder({orderId, productId, price, quantity})`

Reply:
updated and resolved.